lhotse/recipes/csj.py

```diff
@@ -0,0 +1,246 @@
+from glob import glob
+from itertools import islice
+import logging
+from concurrent.futures.thread import ThreadPoolExecutor
+from pathlib import Path
+from typing import Dict, List, Sequence, Tuple, Union
+
+from tqdm.auto import tqdm
+
+from lhotse import validate_recordings_and_supervisions, CutSet
+from lhotse.audio import Recording, RecordingSet
+from lhotse.recipes.utils import manifests_exist, read_manifests_if_cached
+from lhotse.supervision import SupervisionSegment, SupervisionSet
+from lhotse.utils import Pathlike
+
+import random
+
+"""
```

Comment:
I think we usually put this at the top of the file before the imports.

Reply:
done